golang/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.12
 
 require (
 	github.com/cosmos/cosmos-sdk v0.37.0
+	github.com/go-logfmt/logfmt v0.4.0 // indirect
```

Comment:
I wonder why yours keeps adding this back :thinking:

Reply:
mine keeps removing it

Reply:
No idea...